index.html

```diff
@@ -0,0 +1,171 @@
+                    <span class="program-img"><img src="images/icons/workshop.png" alt="workshop"></span>
+                    <h3 class="font-2 color-1">Workshop</h3>
+                    <p class="font-2 color-3 ">
+                        Try out different tech tools that can help you come up with energy conservation measures more easily and in more
+                        efficient ways.
+                    </p>
+                </div>
+                <div class="program">
+                    <span class="program-img"><img src="images/icons/network-icon.png" alt="Network"></span>
+                    <h3 class="font-2 color-1">Network</h3>
+                    <p class="font-2 color-3">
+                        Get the opportunity to network with various people who have experience in development and have
+                        developed outstanding projects
+                    </p>
+                </div>
+            </div>
+            <button type="button" class="join-cc color-3 font-1 mobile-only">Join CC Energy Summit 2022</button>
+            <a href="#" class="desktop-only font-2 color-3">
+                SEE THE WHOLE PROGRAM <i class="fa-solid fa-angle-right"></i>
+            </a>
+        </section>
+        <section class="speakers">
+            <h4 class="font-2 color-2 speaker-header">Featured Speakers</h4>
+            <div class="red-line"></div>
+            <div class="speaker-list">
+            </div>
+            <button type="button" class="color-2 font-2 see-more mobile-only">
+                SEE MORE <i class="fa-solid fa-angle-down color-1"></i>
+            </button>
+        </section>
+        <section class="partners">
+            <h6 class="font-2 color-3">Partners</h6>
+            <div class="red-line"></div>
+            <div class="partner-logo">
+                <img class="" src="images/partners/nike-logo.svg" alt="nike logo">
+                <img class="" src="images/partners/addidas.png" alt="adiddas logo">
+                <img class="" src="images/partners/google-logo.svg" alt="google logo">
+                <img class="" src="images/partners/mozilla.svg" alt="mozilla logo">
+                <img class="" src="images/partners/gucci.svg" alt="gucci logo">
+            </div>
+        </section>
+        <footer>
+            <span class="footer-logo">
+                <img src="images/lg5.png" alt="Logo">
+            </span>
+            <div class="footer-text">
+                <p class="font-2 color-2">
+                    2022 Global Energy summit . All Rights Reserved.
+                </p>
+                <p class="desktop-only font-2 color-2">
+                    All contents on this website, unless stated otherwise, is at the Global Energy shop in Dubai.
+                    <br>For
+                    detailed usage, function details and permission conditions, see https://energysummit.org
+                </p>
+            </div>
+        </footer>
```

Comment:
- [ ] Please, try fixing your footer for the main page of desktop version, because it looks not like an original one and also the logo is not clearly displaying - the logo is distorted due to the size, you can see on the screenshot below:
<img width="1423" alt="Screen Shot 2022-03-15 at 11 55 20" src="https://user-images.githubusercontent.com/49064106/158333038-9c2da268-3168-4377-a95e-dae1367ccbcb.png">

- [ ] Please, consider the alignment of the logo and text, as is shown on the original design below:
<img width="974" alt="Screen Shot 2022-03-15 at 11 55 40" src="https://user-images.githubusercontent.com/49064106/158333202-d57d77da-f5d6-44e2-881a-d9456c88487f.png">